### ISSUE TITLE
DragonAPI - Held items render twice if another player is in the draw area

### ISSUE THREAD

**Issue by charagarlnad** [ReikaKalseki/Reika_Mods_Issues#2183]
This applies to LAN or regular SMP, if more than 1 player is visible on screen at a time, any sort of action gets mirrored to the other players. I.e. 2 players, both in third-person mode will see each others held item movements on a duplicate of their own held item, or, 3 players all in first person will see the same effect. Here's a video demonstrating: https://www.youtube.com/watch?v=7ZVQOn9SKQw

It appears that this issue has existed for quite a while as the only other instance of something similar to  this I could find was a 3 year old post on reddit: https://www.reddit.com/r/feedthebeast/comments/4h18kq/double_item_glitch/

Tested with latest Forge (10.13.4.1614) and only DragonAPI (DragonAPI+1.7.10+V22d).

Just to follow tradition here are logs from the 2 clients, though they have nothing of substance to help - https://gist.github.com/charagarlnad/465a843e809f1868b7add55677c1328d

**Comment by OvermindDL1** [comment]
Oh hey, I noticed that too but had no clue what was causing it.  Why would DragonAPI be screwing with entity-held item rendering, and *where*?!

It happens with **just** DragonAPI and nothing else yes?

**Comment by charagarlnad** [comment]
> Oh hey, I noticed that too but had no clue what was causing it. Why would DragonAPI be screwing with entity-held item rendering, and _where_?!
> 
> It happens with **just** DragonAPI and nothing else yes?

Yeah, a fresh forge install with just DragonAPI does it. It's more than just items as wearing skulls can cause the same effect with 2 different copies of it on the player too.

**Comment by ReikaKalseki** [comment]
I have never seen this.

**Comment by OvermindDL1** [comment]
I've seen this, but didn't know what mod it was, but it was when I had DragonAPI installed.  I generally only play multiplayer and if this doesn't happen in singleplayer...

**Comment by ReikaKalseki** [comment]
I generally played multiplayer as well.

**Comment by charagarlnad** [comment]
I've seen this problem for at least a year but didn't try to narrow it down until now, I only play multiplayer too but it's pretty easy to gloss over. I've been able to get people on my server to reproduce it across many different environments; Windows & Linux, Hotspot & OpenJ9, Intel & AMD, etc. so I don't believe its a rare localized issue. Here's a screenshot of how it works in 1st person too: ![Screenshot](https://user-images.githubusercontent.com/27380203/64656945-48063b80-d3ff-11e9-9776-6f4b1f970381.png) Note that there are other mods here but it has been reproduced with just DragonAPI in the first video.

**Comment by waterlubber** [comment]
I also experience duplicate items in CrC item stands -- they will often render two item entities, mirrored from each other, when I place an item into the stand.
